FirebaseABTesting/Sources/Public/ABTExperimentPayload.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google
```

Comment:
If we're not expanding the public API, this file should be put in Sources/Private and added to s.private_header_files in the podspec.

Reply:
Done in 3542804.